packages/esm-patient-search-app/src/index.ts

```diff
@@ -35,6 +36,8 @@ export const patientSearchBar = getSyncLifecycle(patientSearchBarComponent, opti
 export function startupApp() {
   defineConfigSchema(moduleName, configSchema);
 
+  registerFeatureFlag('mpiFlag', 'MPI Service', 'Enables the Master Patient Index workflows');
```

Comment:
Probably should've said this: our preferred means of registering feature flags is via `routes.json`. Basically add the same properties, but to a `featureFlag` definition there.